packs/remedy/sensors/workorder_sensor.py

```diff
@@ -0,0 +1,71 @@
+from st2reactor.sensor import PollingSensor
+from actions.lib.actions import BaseAction
+from actions import *
+
+class SimplePollingSensor(PollingSensor):

```

Comment:
Needs to match the value in https://github.com/StackStorm/st2contrib/pull/333/files#diff-9ffe7287116d697cc2b8d56b8217cceeR2.

Therefore use `WorkOrderSensor`